openapiart/bundler.py

```diff
@@ -316,8 +323,13 @@ def _apply_common_x_field_pattern_properties(self, schema, xpattern, format, pro
         if format is not None:
             schema["format"] = format
         if "length" in xpattern:
-            schema["minimum"] = 0
-            schema["maximum"] = 2 ** int(xpattern["length"]) - 1
+            if "format" in xpattern and xpattern["format"] in ["ipv4", "ipv6", "mac"]:
+                self._errors.append("property %s should not contain length as format set to %s" %(
```

Comment:
The error should be more specific to indicate all the keywords that are not allowed such as minLength, maxLength, minimum, maximum.